packages/typestack/src/validate-body.hook.ts

```diff
@@ -38,5 +39,5 @@ export function ValidateBody(
 }
 
 function isArrowFunction(value: any): value is ((controller: any) => Class) {
-  return !value.hasOwnProperty('prototype');
+  return !Object.prototype.hasOwnProperty.call(value, 'prototype');
```

Comment:
Why this change?

Reply:
Just to be sure that we won't break, if someone gives us object with builtin properties messed up. Maybe it's overkill